src/packet.rs

```diff
@@ -166,6 +178,12 @@ impl<'a> AsRef<[u8]> for ConnectionId<'a> {
     }
 }
 
+impl<'a> std::hash::Hash for ConnectionId<'a> {
+    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
```

Comment:
Nit: use `#[inline]` for small functions in public API. Rust doesn't inline functions without it during cross-crate compilation even if LTO is enabled.

Reply:
We never do `#[inline]` right now, so I'll go through over the public API and do this in a separate PR.

Reply:
https://github.com/cloudflare/quiche/pull/751